src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -109,6 +122,52 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    bool ProfileViewModel::UseParentProcessDirectory()
+    {
+        return StartingDirectory().empty();
+    }
+
+    // This function simply returns the opposite of UseParentProcessDirectory.
+    // We bind the 'IsEnabled' parameters of the textbox and browse button
+    // to this because it needs to be the reverse of UseParentProcessDirectory
+    // but we don't want to create a whole new converter for inverting a boolean
+    bool ProfileViewModel::UseCustomStartingDirectory()
```

Comment:
Would like thoughts on this - I wanted to avoid creating a convertor to invert a bool so I made this function (the checkbox binds to `UseParentProcessDirectory` and the textbox binds to this one which is simply the inverse)

Would we prefer a converter instead?

Reply:
I'm fine with this property value honestly.

Reply:
It's less work than a converter.